scripts/eval/yamls/mpt_eval.yaml

```diff
@@ -1,44 +1,43 @@
 max_seq_len: 2048
 tokenizer_name: EleutherAI/gpt-neox-20b
 seed: 1
-precision: fp32
+precision: amp_fp16
 
-# Tokenizer
-tokenizer:
-  name: ${tokenizer_name}
-  kwargs:
-    model_max_length: ${max_seq_len}
 
-model:
-  name: mpt_causal_lm
-  init_device: meta
-  d_model: 768
-  n_heads: 12
-  n_layers: 12
-  expansion_ratio: 4
-  max_seq_len: ${max_seq_len}
-  vocab_size: 50368
-  attn_config:
-    attn_impl: triton
+models:
+-
+  model_name: mpt_test
+  # Tokenizer
+  tokenizer:
+    name: ${tokenizer_name}
+    kwargs:
+      model_max_length: ${max_seq_len}
+  model:
+    name: mpt_causal_lm
+    init_device: meta
```

Comment:
this kind of breaks the mold of how we structure our yamls.
Why do this nesting?